Druid/DRBytecodeGenerator.class.st

```diff
@@ -184,7 +215,8 @@ DRBytecodeGenerator >> visitNullValue: aDRNullValue [
 DRBytecodeGenerator >> visitPhiFunction: aDRPhiFunction [
 
 	| resolved |
-	resolved := aDRPhiFunction valuesForPredecessor: predecessor.
+	resolved := aDRPhiFunction valuesForPredecessor: (self visitedPredecessorOf: aDRPhiFunction basicBlock).
```

Comment:
Well, the thing is that with reverse post order before we were always visiting the predecessors before visiting a block :)
That could simplify a bit this

Reply:
Ahh yes, this class implement the old algorithm. Now Treegraph is used, so probably we will remove this class.